cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,131 @@
+/// <reference types="Cypress"/>
+
+describe('EngageSphere', () => {
+    beforeEach(() => {
+
+        cy.visit('')
+        
+    })
+
+    it('Valida se H1 esta visivel e alterna tema', () => {
+        cy.contains('h1', 'EngageSphere').should('be.visible')
+        cy.get('#theme-toggle-button').click()
+    })
+
+    it.only('Valida mensagem de saudação', () => {
+        const date = new Date(Date.UTC(2024, 8, 22))
+        cy.clock(date)
+        cy.contains('p', 'Hi there! It is now Sat Sep 21 2024').should('be.visible')
+    });
+
+    it.only('Valida saudação personalizada', () => {
+        const date = new Date(Date.UTC(2024, 8, 22))
+        cy.clock(date)
+        cy.get('input#name').type('Félix')
+        cy.contains('p', 'Hi Félix! It is now Sat Sep 21 2024').should('be.visible')
+    
+    });
+
+    it('Muda o tema para modo escuro, garantindo que persista no armazenamento local', () => {
+        cy.get('button#theme-toggle-button').click()
+
+        cy.window().then((window) => {
+            const theme = window.localStorage.getItem("theme")
+            expect(theme).to.eq('dark')
+        })
+    });
+
+    it('Muda o tema para modo claro, garantindo que persista no armazenamento local', () => {
+        cy.get('button#theme-toggle-button').click()
+        cy.window().then((window) => {
+            const theme = window.localStorage.getItem("theme")
+            expect(theme).to.eq('dark')
+        })
+        cy.get('button#theme-toggle-button').click()
+        cy.window().then((window) => {
+            const theme = window.localStorage.getItem("theme")
+            expect(theme).to.eq('light')
+        })
+    })
+    it('Valida se o campo de texto esta desabilitado ', () => {
```

Comment:
```suggestion
    it('Valida se o campo de texto esta desabilitado', () => {
```

Evite espaços em branco desnecessários.